src/dex-helper/dummy-dex-helper.ts

```diff
@@ -100,9 +100,9 @@ class DummyBlockManager implements IBlockManager {
     contractAddress: Address | Address[],
     afterBlockNumber: number,
   ): void {
-    console.log(
-      `Subscribed to logs ${subscriber.name} ${contractAddress} ${afterBlockNumber}`,
-    );
+    // console.log(
+    //   `Subscribed to logs ${subscriber.name} ${contractAddress} ${afterBlockNumber}`,
+    // );
```

Comment:
These messages were quote useful to understand that subscribe to logs was called in tests

Reply:
Was carried over from the previous version that subscribed to all pools at startup (no longer doing that), I have re-enabled this logging now.